### ISSUE TITLE
Class prefixes on iOS

### ISSUE THREAD

**Issue by brentvatne** [wix/react-native-interactable#37]
[It's common practice for iOS libraries to include some kind of class prefix for name spacing](http://nshipster.com/namespacing/), eg: on React Native, `RCTx`. Maybe `WIXx` here?

**Comment by talkol** [comment]
You're right. Since I'm aiming eventually to PR into the core (once the issues are ironed out and people are happy with the API), what do you think about using the core namespace? (RCT)

**Comment by brentvatne** [comment]
Somehow I totally missed that reply. It's probably best to use `WIX` or something along those lines for now, it can always be changed if it gets into core.

**Comment by sonaye** [comment]
@talkol There is currently an [interest](https://expo.canny.io/feature-requests/p/add-wix-interactable-library) to bring Interactable to Expo, @brentvatne is leading the efforts I believe, this prefixes issue is what is blocking the library from being included as a part of the Expo SDK. I am not sure on how difficult it is to make the changes needed, I would submit a PR if I could, but I am not an expert on the matter. Just wanted to point out that adding Interactable to Expo would not only benefit the library itself, but also it adds a great value to those who are developing on top of Expo.

**Comment by salzhrani** [comment]
If the issue needs just renaming ios files and references, I am glad to try a PR.

**Comment by rotemmiz** [comment]
@brentvatne, is there still such plan? is it already being used? We'd be happy to collaborate!

**Comment by brentvatne** [comment]
hey @rotemmiz! I think now this library isn't needed because you can do the same thing with react-native-gesture-handler and react-native-reanimated as demonstrated by this re-implementation on top of those libraries: https://github.com/kmagiera/react-native-reanimated/tree/master/Example/interactablePlayground
